src/ChipInput.js

```diff
@@ -508,6 +508,7 @@ class ChipInput extends React.Component {
               return chipRenderer({
                 value,
                 text: dataSourceConfig ? tag[dataSourceConfig.text] : tag,
+                chip:tag,
```

Comment:
Please just add a space here (e.g. lint the code), then it's fine. :+1: 
Also, please update the README accordingly.